features/fetch.yml

```diff
@@ -10,34 +10,54 @@ compat_features:
   - api.Headers.append
   - api.Headers.delete
   - api.Headers.get
+  - api.Headers.getSetCookie
   - api.Headers.has
   - api.Headers.set
   - api.Request
   - api.Request.Request
+  - api.Request.Request.init_referrer_parameter
   - api.Request.arrayBuffer
   - api.Request.blob
+  - api.Request.body
   - api.Request.bodyUsed
+  - api.Request.bytes
+  - api.Request.cache
   - api.Request.clone
   - api.Request.credentials
+  - api.Request.destination
+  - api.Request.formData
+  - api.Request.integrity
+  - api.Request.isHistoryNavigation
   - api.Request.headers
   - api.Request.json
+  - api.Request.keepalive
   - api.Request.method
   - api.Request.mode
+  - api.Request.redirect
+  - api.Request.referrer
+  - api.Request.referrerPolicy
   - api.Request.text
   - api.Request.url
   - api.Response
   - api.Response.Response
   - api.Response.arrayBuffer
   - api.Response.blob
   - api.Response.bodyUsed
+  - api.Response.body
+  - api.Response.bytes
   - api.Response.clone
+  - api.Response.error_static
+  - api.Response.formData
   - api.Response.headers
   - api.Response.json
   - api.Response.json_static
   - api.Response.ok
+  - api.Response.redirect_static
+  - api.Response.redirected
   - api.Response.status
   - api.Response.statusText
   - api.Response.text
   - api.Response.type
   - api.Response.url
   - api.fetch
+  - api.fetch.authorization_removed_cross_origin
```

Comment:
Should these be considered?

- api.fetch.blob_data_support
- api.fetch.init_attributionReporting_parameter
- api.fetch.init_keepalive_parameter
- api.fetch.init_referrerPolicy_parameter